x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/PutDataFrameTransformAction.java

```diff
@@ -47,9 +46,10 @@ private PutDataFrameTransformAction() {
         return AcknowledgedResponse::new;
     }
 
-    public static class Request extends AcknowledgedRequest<Request> implements ToXContentObject {
+    public static class Request extends AcknowledgedRequest<Request> {
 
         private final DataFrameTransformConfig config;
+        private boolean deferValidation = false;
```

Comment:
I think this could be made final if you added constructor parameter "deferValidation" and removed setter method.